src/main/java/uk/gov/di/ipv/cri/common/library/domain/DeviceInformation.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.di.ipv.cri.common.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DeviceInformation {
```

Comment:
For safety, might be worth adding `@JsonIgnoreProperties(ignoreUnknown = true)` as well - if any extra/unknown fields come in, Jackson will throw an exception. Adding the `@JsonIgnoreProperties` annotation will stop any exceptions from being thrown. 

You can ignore this comment, if we want to throw.

Reply:
Hmm yeah I think we want to throw in that case - this is being put directly into an audit event so we have pretty set schemas for that but good to know for future 👍